libs/ui/scripts/finalize_app_strings_catalog.ts

```diff
@@ -0,0 +1,39 @@
+import fs from 'fs';
+
+import {
+  UiStringTranslations,
+  UiStringTranslationsSchema,
+  safeParseJson,
+} from '@votingworks/types';
+import { generateNumberStringsCatalog } from '../src/ui_strings/number_strings';
+
+const APP_STRINGS_CATALOG_FILE_PATH =
+  './src/ui_strings/app_strings_catalog/latest.json';
```

Comment:
This path makes it such that this script has to be run from the base of `libs/ui`, which is fine since we'll almost always be running this through `pnpm build:app-strings-catalog`. But for flexibility, could use `__dirname` here, e.g. `path.join(__dirname, '../src/ui_strings/app_strings_catalog/latest.json')`

Reply:
ah, thanks - good call